package.json

```diff
@@ -38,7 +38,7 @@
     "test:e2e:all": "yarn test:e2e:clean && yarn test:e2e; yarn test:e2e:report;",
     "test:e2e:kind": "CYPRESS_dexUrl=http://dex-twodotoh.a.influxcloud.dev.local CYPRESS_baseUrl=https://twodotoh.a.influxcloud.dev.local CYPRESS_password=password cypress open --browser chrome --config chromeWebSecurity=false",
     "test:e2e:remocal": "CYPRESS_dexUrl=https://dex-${NS}.remocal.influxdev.co CYPRESS_baseUrl=https://${NS}.remocal.influxdev.co CYPRESS_password=password cypress open --browser chrome --config chromeWebSecurity=false",
-    "test:circleci": "yarn run test:ci --maxWorkers=2",
+    "test:circleci": "yarn run test:ci --runInBand",
```

Comment:
Similar question to above - why do we want tests running serially here? https://jestjs.io/docs/cli#--runinband

Reply:
From what I can see, the unit test times went from [<3min to 4-6minutes](https://app.circleci.com/pipelines/github/influxdata/ui?branch=bucky_upgrade_webpack) the moment that [the commit went in](https://github.com/influxdata/ui/pull/6319/commits/081062727a3708d70b3dd6f5ee105ce39fcdf7a4) to change these flags. This is not a long time, especially given how long monitor-ci takes.

However, I'm not sure if we need this change.

I would note that when this change/commit went in, the CI unit test began passing instead of the [previous failure](https://app.circleci.com/pipelines/github/influxdata/ui/18960/workflows/06663776-00be-4648-9a75-6be64c2c6e86/jobs/105184). However, the previous failure was in only the variables unit test -- which is now skipped by another commit pushed at the same time. Therefore, I think it may be worthwhile to see if we really need this change?

Reply:
@wdoconnell @wiedld copied from this comment: https://github.com/influxdata/ui/pull/6319/files#r1037614697

it turns out that our jest tests were running out of memory. The general way the internet recommends fixing it is to set `maxWorkers` <= the number of CPUs your circle instance has and go from there. Since setting `maxWorkers` to 2 was causing problems, I dropped the number down to `1`, which is an alias for `runInBand`.

I talked with Gene about this. This isn't necessarily the best way to do it - there are tradeoffs. Lowering the number of CPUs will make the tests run for longer which will cost more money, but we'll still be on a relatively cheap CPU plan. The alternative is to increase the number of CPUs for this circleci job which will also cost more money. If this ends up taking too long or being too expensive, we can increase the resources available to our circle instance.